index.html

```diff
@@ -0,0 +1,196 @@
+                <img src="./img/bookart.jpg" alt="first program">
+            </div>
+            <h3>LECTURER</h3>
+            <p>A modern and contemporary art collection with local presence in “Villa Lakeside”.</p>
+          </div>
+          <div class="program-item">
+            <div class="program-item-icon">
+                <img src="./img/art1.jpg" alt="second program">
+            </div>
+            <h3>Exhibition</h3>
+            <p>A modern and contemporary art collection with local presence in “Villa Lakeside”.</p>
+          </div>
+          <div class="program-item">
+            <div class="program-item-icon">
+                <img src="./img/artevent.png" alt="third program">
+            </div>
+            <h3>Art Forum</h3>
+            <p>A modern and contemporary art collection with local presence in “Villa Lakeside”.</p>
+          </div>
+          <div class="program-item">
+            <div class="program-item-icon">
+              <img src="./img/art9.jpg" alt="fith program">
+            </div>
+            <h3>workshop</h3>
+            <p>A modern and contemporary art collection with local presence in “Villa Lakeside”.</p>
+          </div>
+          <div class="program-item">
+            <div class="program-item-icon">
+                <img src="./img/art3.jpg" alt="sixth program">
+            </div>
+            <h3>Art teaching</h3>
+            <p>A modern and contemporary art collection with local presence in “Villa Lakeside”.</p>
+          </div>
+          <input type="button" class="program-btn" value="See more" />
+        </div>
+        <a href="#" class="program-link"> SEE THE WHOLE PROGRAM </a>
+      </section>
+
+
+
+      <section class="row d-flex justify-content-center">
+          
+        <div class="ranking-section container-fluid py-5">
+          <h2 class="container text-cener pt-5 d-flex justify-content-center">Feature Artists</h2>
+          <div class="container d-flex my-3 justify-content-center cont-nav"></div>
+        </div>
+      
+      </section>
+     
+<section class="hide">
+      <section class="partners">
+        <h2>Partners</h2>
+        <div class="partners-underline"></div>
+        <div class="partners-logo">
+            <img class="ptners-logo" src="./img/logogray.png" alt="">
+            <img class="ptners-logo" src="./img/logogray4.png" alt="">
+            <img class="ptners-logo" src="./img/logogray.png" alt="">
+            <img class="ptners-logo" src="./img/logogray4.png" alt="">
+        </div>
+      </section>
+      <footer>
+        <div class="footer-container">
+            <div class="col-md-3 py-5 text-center footer-title">© ART EXHIBITION</div>
+            <div class="col-md-9 p-5">
+                <h5 class="pb-3 mt-3 footer-params">2022 - Art exhibitions represent the gathering of art objects into a space for a temporary event. Some rights reserved </h5>
+  
+              </div>
+        </div>
+      </footer>
```

Comment:
- [ ] Great job in hiding this section as requested by the [previous reviewer](https://github.com/GloireIrenge/Art-exhibition-project/pull/1#discussion_r930685171). I have realized that there is a white background that is showing in its place which was not there originally. Kindly fix this.

**This is what the section looks like on my end:**

![image](https://user-images.githubusercontent.com/86318284/181688947-905d0474-a624-4d4b-bb52-58b89cffdd1a.png)

Reply:
Hey @GloireIrenge, this issue @Nemwel-Boniface talked about in the previous review still persists. Have a look at this screenshot. It has to do something with `section` with the class of `hide`. Look into it and kindly fix it.
![image](https://user-images.githubusercontent.com/33268927/181700606-850a5dc8-c030-434e-bf05-20ef3ffa5eb4.png)